.github/workflows/scripts/common/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .bootstrap import init
+init()
```

Comment:
ditto: let's wrap this in the main so that it's friendly for future imports from other scripts ;)

Reply:
This have to be here since it is doing some very early initialization (mainly automatically installing missing dependencies), put it into `main` would defeat its purpose.

Reply:
oops my bad, this one doesn't matter at all :D